blocks/quote/quote.css

```diff
@@ -1,42 +1,46 @@
 .quote blockquote {
-  margin: 0 auto;
-  padding: 0 1.5rem;
-  max-width: 48rem;
+  --color-text: var(--spectrum-gray-700);
+  --color-border: var(--spectrum-gray-400);
+  position: relative;
+  margin: var(--spectrum-spacing-800) auto;
+  padding-block: 4.5rem;
+  font-family: var(--spectrum-serif-font-family);
+  line-height: 1.4;
+  letter-spacing: 0;
+  color: var(--color-text);
+  border-block: 1px solid var(--color-border);
 }
 
-@media (min-width: 48rem) {
-  /* !! min-width MQ needs refactor */
-  .quote blockquote {
-    padding: 0 2rem;
-  }
+.quote blockquote p {
+  font-size: 1.75rem;
+  margin: 0;
 }
 
-.quote blockquote .quote-quotation {
-  font-size: 120%;
-}
-
-.quote blockquote .quote-quotation > :first-child {
-  text-indent: -0.5ch;
-}
-
-.quote blockquote .quote-quotation > :first-child::before,
-.quote blockquote .quote-quotation > :last-child::after {
-  line-height: 0;
-}
-
-.quote blockquote .quote-quotation > :first-child::before {
-  content: "“";
-}
-
-.quote blockquote .quote-quotation > :last-child::after {
-  content: "”";
+@media (min-width: 48rem) {
+  .quote blockquote p {
+    font-size: 2.25rem;
+  }
 }
 
 .quote blockquote .quote-attribution {
-  text-align: right;
-}
+  margin-block-start: 0.5rem;
+
+  & p {
```

Comment:
I don't think this utility class existed when you wrote this, but for the attribution text, we can give it the `.util-detail-s` class in `quote.js`!

Reply:
Perfect!